sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectKey=DFE-Digital_early-careers-framework
+sonar.organization=dfe-digital
+sonar.exclusions=
+sonar.sources=app,lib
+sonar.tests=./spec
+sonar.ruby.coverage.framework=RSpec
+sonar.ruby.rubocop.reportPath=rubocop-result.json
```

Comment:
do we need the last two lines, I think they ended up being removed from NPQ?

Reply:
ah yeah we override these in the sonarscanner CLI call anyway, I'll remove 👍